### ISSUE TITLE
expand/collapse icons showing as weird unicode characters

### ISSUE THREAD

**Issue by joachim-n** [bhollis/jsonview#218]
**Describe the bug**

The collapse/expand icons are showing as unicode characters. See screenshot:

<img width="612" alt="Screenshot 2024-08-12 at 15 06 31" src="https://github.com/user-attachments/assets/17357d19-6668-400f-a359-ddb9a18d275f">

**What browser are you using?***

Firefox 129.0 on MacOS 13.5.1

**Where did you install JSONView from?**
No idea, sorry. Firefox add-ons says JSONView 3.0.1 with homepage https://jsonview.com/.

**Comment by UWesthaus** [comment]
I have the same issue since a long time on Firefox 129.0.2 (64 bit) on Windows 10.

The error is here: moz-extension://cb57a05e-6369-4338-86f0-180e62f5dfca/viewer.css

![grafik](https://github.com/user-attachments/assets/8c75fff0-6207-4aac-afbd-214417d51eda)

But I am unable to find or correct the file.

**Comment by stanio** [comment]
> The error is here: moz-extension://cb57a05e-6369-4338-86f0-180e62f5dfca/viewer.css

It is not the only/main issue.  I've built a patched version, adding at the top of `viwer.css`:

```css
@charset "UTF-8";
```

This fixes the expand/collapse icons, but non-ASCII content is still garbled (note the `japanese` value):

Bad | CSS-fix | Good
---|---|---
![Bad](https://github.com/user-attachments/assets/e1e19986-774e-4374-8a09-28cf161a6d56) | ![CSS-fix](https://github.com/user-attachments/assets/c2c6ce5d-9000-4104-bd48-832d1b81b4ea) | ![Good](https://github.com/user-attachments/assets/d929dc90-b068-4726-842a-619d567e4420)

All shots use [`example.json`](https://jsonview.com/example.json) downloaded and served from a local web server.  Both are served with:

```http
Content-type: application/json
```

The "Good" is the original `example.json` that appears to start with UTF-8 BOM – this one results in correct expand/collapse icons and content w/o my extension patch.  In the "Bad" and "CSS-fix" ones, I have removed the UTF-8 BOM from the file content.  The latter has the suggested CSS fix applied to the extension.

**Comment by stanio** [comment]
> In the "Bad" and "CSS-fix" ones, I have removed the UTF-8 BOM from the file content.

Funny enough, when I open the file via `file:` it shows just fine.

**Comment by stanio** [comment]
> The content type should have `charset` added when changed to `text/*`...

Also, changing the content type appears unnecessary when the Firefox built-in viewer is disabled explicitly by the user (`about:config`):

```
devtools.jsonview.enabled = false
```

**Comment by stanio** [comment]
Fixed for me using JSONView 3.0.2 in Firefox.
